src/events.ts

```diff
@@ -44,15 +56,64 @@ type ExitRequestedEvent = {
 
 type DreamsEvent = IdTokenDidExpireEvent |
                    AccountProvisionRequestedEvent |
+                   InvestmentAccountProvisionRequestedEvent |
                    ExitRequestedEvent |
                    ShareEvent;
 
-export default messages;
+type AccountProvisionInitiatedEvent = {
+  name: partnerEvents.accountProvisionInitiated;
+  message: {
+    requestId: string;
+  }
+}
+
+type InvestmentAccountProvisionInitiatedEvent = {
+  name: partnerEvents.investmentAccountProvisionInitiated;
+  message: {
+    requestId: string;
+    dreamId?: string;
```

Comment:
:trollface:

Reply:
duh! I'm going to move from js tests to ts tests (but as a separate effort - this ticket is a bit out of hand already)